TidepoolKit Example/Support/Extensions/Sample.swift

```diff
@@ -64,6 +110,21 @@ struct Sample {
                                         energy: TFoodDatum.Nutrition.Energy(350, .kilocalories))
         }
 
+        fileprivate static var cgmSettingsAlerts: TCGMSettingsDatum.Alerts {
+            let snooze = TCGMSettingsDatum.Alerts.Snooze(30, .minutes)
+            return TCGMSettingsDatum.Alerts(enabled: true,
```

Comment:
Is this intended to only represent the configurable alerts?  I ask since the only configurable part of the `urgentLow` alert (maybe the `urgentLowPredicted` alert too, but I haven't implemented this in Tidepool Loop) is the sound. I don't see any sounds included here. So makes me wonder if `urgentLow` should be included. If yes, then should non-configured alerts also be included? It is totally possible various CGMs will have various non-configurable alerts. Do we need to track these alerts such that we can recreate a specific scenario?

Reply:
The `TCGMSettingsDatum` represents the *current* Tidepool Data Model and API and was mostly modeled after the Dexcom API. Specifically, this endpoint: https://developer.dexcom.com/get-devices

I have not yet made any changes that might be needed to represent Tidepool Loop Dexcom CGM settings. We should start a conversation on how to get those CGM settings into `Loop`/`LoopKit` proper and then to `TidepoolService` and `TidepoolKit` so we can figure out what, if any, changes we need to make to the Tidepool Data Model and API. (As far as I am aware, the Dexcom G6 CGM settings are still "private" to the plugin.)

In response to your specific questions, this represents what the Dexcom API sends to Tidepool backend. If I understand correctly, it does represent that latest configuration (not defaults) and includes user-settable (e.g. high alert) and fixed alerts (e.g. urgent low).

In our future conversation, we should discuss user-settable and fixed alerts as well as sounds.

Another thing we should consider is whether the settings of each plugin are part of the official documented Tidepool Data Model and API or whether we have a "private black box" data blob for saving and restoring this configuration.